public/index.html

```diff
@@ -42,4 +41,6 @@
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
   </body>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/3.5.1/gsap.min.js"></script>
```

Comment:
Why do you use CDN when already installed the `npm` package ?

Reply:
It is for another implementation entirely. Removing it might have skipped my mind.